embulk-ruby/lib/embulk/java/imports.rb

```diff
@@ -1,4 +1,14 @@
 require 'java'
+#
+# This statement is necessary for using JRuby.runtime method from JRuby 9.3.0.0.
+# JRuby.runtime happened to work without `require jruby` < 9.3,
+# because it was implemented by Java core feature.
+# This implementation has been changed in JRuby 9.3.0.0.
+#
+# See also.
+# https://github.com/jruby/jruby/issues/7221#issuecomment-1133646241
+#
+require 'jruby' if JRUBY_VERSION.to_f >= 9.3
```

Comment:
Reading the comment in `jruby/jruby`, how about the following?

At the same time, `to_f >= 9.3` does not seem safe to me.

```suggestion
# Required to use "Java features" from the Ruby side.
require 'java'

# Required to access JRuby-specific internal features, such as `JRuby.runtime`.
require 'jruby' if Gem::Version.new(JRUBY_VERSION) >= Gem::Version.new('9.3')

# Both of 'java' and 'jruby' should be loaded in Embulk's use-case in nature.
#
# Embulk had loaded only 'java' explicitly until Embulk v0.10.35.
# It was because loading 'java' automatically loaded also 'jruby' until JRuby 9.2.
# But, it has changed since JRuby 9.3. JRuby 9.3+ needs loading 'jruby' explicitly.
#
# See also: https://github.com/jruby/jruby/issues/7221#issuecomment-1133646241
#
# Here, Embulk limits loading 'jruby' only in JRuby 9.3+ to ensure compatibility.
# We may remove the limitation in the future.
```